dynamic-scoring-framework/samples/ai-workload-scorer/app/main.py

```
@@ -0,0 +1,349 @@
+
+    cluster_base_forecast = np.sum(list(node_base_forecasts.values()))
+    print("Cluster base forecast:", cluster_base_forecast)
+
+    # Individual app-device level power aggregation and scoring
+    for dim in dimentions:
+        node_histories = []
+        for node_label, df_node in dfs_node.items():
+            relevant_cols = dim_relevant_cols[node_label][dim]
+            if not relevant_cols:
+                continue
+            df_subset = df_node[["timestamp"] + relevant_cols]
```

Ruff (0.15.9)

[warning] Consider `["timestamp", *relevant_cols]` instead of concatenation

Replace with `["timestamp", *relevant_cols]`

(RUF005)